robottelo/host_helpers/satellite_mixins.py

```diff
@@ -39,6 +39,18 @@ def enable_puppet_satellite(self):
         assert 'Success!' in result.stdout
         return self
 
+    def enable_multicv_setting(self):
+        cfg_file = 'upstream_only_settings.rb'
+        cfg_path = self.execute(f'find /usr/share/gems/gems/ -name {cfg_file}').stdout.strip()
+        assert cfg_file in cfg_path, 'Config file not found'
+        self.execute(
+            f'sed -i "s/allow_multiple_content_views/#allow_multiple_content_views/g" {cfg_path}'
+        )
+        self.cli.Service.restart()
+        assert len(
+            self.api.Setting().search(query={'search': f'name={"allow_multiple_content_views"}'})
+        ), 'Multi-CV enablement failed'
+
```

Comment:
what are we checking here exactly, could you explain?

Reply:
We are checking that after rb file manipulation and service restart the setting became available.

Reply:
I mean with this assertion, if setting exists or not iiuc? if yes, I'd suggest adding a comment which could help understand this fixture